lib/db.js

```diff
@@ -206,6 +211,14 @@ function Db(databaseName, topology, options) {
   topology.on('reconnect', createListener(this, 'reconnect', this));
 }
 
+Object.defineProperty(Db.prototype, 'optionsValidationLevel', {
+  enumerable: true,
+  get: function() {
+    if (this.s && this.s.optionsValidationLevel) return this.s.optionsValidationLevel;
+    return this.s.client.optionsValidationLevel;
```

Comment:
unfortunately I don't believe we currently store `client` in `this.s`. Consequently I think this shows that we don't (yet) have tests for this new inherited property approach